docs/docs/how-tos/nebari-destroy.md

```diff
@@ -0,0 +1,180 @@
+
+:::warning
+When working with cloud resources, it's important to always double check that everything has been fully destroyed.
+
+Further below, we will also take a look at how you can manually destroy resources on individual cloud providers in case the automation fails.
+
+The Nebari team takes no responsibility for any incurred cloud costs. Please take care!
+:::
+
+## Destroying Nebari
+
+You can specify the `nebari-config.yaml` configuration file created while deploying Nebari to destroy the resources as well. Type the following command on your command line:
+
+:::warning
+If you have any data stored on your Nebari cluster you would like to keep - either files on the filesystem, conda environments in conda-store or an exported `json` of the users and groups from Keycloak - now is the time to back those up **off of the cluster**. 
+:::
+
+```bash
+nebari destroy -h
+```
+
+```bash
+ Usage: nebari destroy [OPTIONS]
+
+ Destroy the Nebari cluster from your nebari-config.yaml file.
+
+╭─ Options ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ *  --config          -c      TEXT  qhub configuration file path [default: None] [required]                                │
+│    --output          -o      TEXT  output directory [default: ./]                                                         │
+│    --disable-render                Disable auto-rendering before destroy                                                  │
+│    --disable-prompt                Destroy entire Nebari cluster without confirmation request. Suggested for CI use.      │
+│    --help            -h            Show this message and exit.                                                            │
+╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+:::note
+The above command will not delete your `nebar-config.yaml` or the rendered files thus a re-deployment with `nebari deploy` is possible afterwards. 
+:::
+
+Once you've decided to destroy your cluster and have read through the `nebari destroy --help` command, you are you now ready.
+
+```bash
+nebari destroy -c nebari-config.yaml
+```
+
+The terminal will prompt you to press `enter` to verify that you wish to destroy your Nebari cluster.
+
+:::info
+This destruction process can take up to 30 minutes for some cloud providers.
+:::
+
+If the destruction is successful, you will see the following output:
+
+```bash
+[terraform]: Nebari properly destroyed all resources without error
+```
+
+Congratulations! You have successfully destroyed your Nebari deployment and the associated resources!
+
+## Manually destroying the resources
+
+### Amazon Web Services (AWS)
+
+Unfortunately of all the cloud providers supported, AWS seems to make it the most difficult to destroy lingering resources. They often times requiring resources be destroyed in a particular order as well.
```

Comment:
```suggestion
:::note
Destroying lingering resources on AWS can be tricky because the resources often need to be destroyed in a particular order.
:::
```

The reasoning behind this nitpick: AWS is a very popular cloud provider, and I think we may scare users by saying it's difficult.

Feel free to modify this suggestion as you see fit!

Reply:
Works for me :)